metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1410,9 +1410,14 @@ class MetalsLanguageServer(
    */
   private def fileWatchFilter(path: Path): Boolean = {
     val abs = AbsolutePath(path)
-    abs.isScalaOrJava || abs.isSemanticdb || abs.isBuild || abs.isBsp
```

Comment:
[nit]
Looks like `isBsp` is no longer used and we can remove :)

Reply:
Removed